### PR TITLE
feat: add notification section under community

### PR DESCRIPTION
**Pull request by ArshdeepGrover**

### CODE REVIEW

**Comment by ArshdeepGrover** [comment]
image


image

File: projects/commudle-admin/src/app/components/sidebar-menu/sidebar-menu.component.ts [hunk 1]
```diff
@@ -88,9 +91,7 @@ export class SidebarMenuComponent implements OnInit {
   getManagingCommunities(userRoles: string[]): void {
     this.managedCommunities = [];
     for (const role of userRoles) {
-      this.communitiesService.getRoleCommunities(role).subscribe((data: ICommunities) => {
-        this.managedCommunities = [...this.managedCommunities, ...data.communities];
-      });
+      this.communitiesService.getRoleCommunities(role).subscribe((data: ICommunities) => {});
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/sidebar-menu/sidebar-menu.component.ts]: no function needed inside the subscribe method

File: projects/commudle-admin/src/app/feature-modules/community-control-panel/community-control-panel-routing.module.ts [hunk 2]
```diff
@@ -32,6 +33,10 @@ const routes = [
         path: '',
         component: CommunityEventsListComponent,
       },
+      {
+        path: 'notification',
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/community-control-panel/community-control-panel-routing.module.ts]: notifications

File: projects/commudle-admin/src/app/feature-modules/community-control-panel/components/community-control-panel/community-control-panel.component.ts [hunk 3]
```diff
@@ -20,6 +20,10 @@ export class CommunityControlPanelComponent implements OnInit, OnDestroy {
       title: 'Events',
       route: `./`,
     },
+    {
+      title: 'Notification',
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/community-control-panel/components/community-control-panel/community-control-panel.component.ts]: notifications

File: projects/commudle-admin/src/app/feature-modules/community-control-panel/components/community-control-panel/community-control-panel.component.ts [hunk 4]
```diff
@@ -20,6 +20,10 @@ export class CommunityControlPanelComponent implements OnInit, OnDestroy {
...
+      route: `./notification`,
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/community-control-panel/components/community-control-panel/community-control-panel.component.ts]: notifications

File: projects/commudle-admin/src/app/feature-modules/notifications/services/notification.service.ts [hunk 5]
```diff
@@ -17,6 +17,13 @@ export class NotificationService {
     return this.http.get<INotifications>(this.apiRoutesService.getRoute(API_ROUTES.NOTIFICATIONS.INDEX), { params });
   }
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/notifications/services/notification.service.ts:19]: rename to notifications.service

File: projects/commudle-admin/src/app/feature-modules/public-community/components/home-community/home-community.component.ts [hunk 6]
```diff
@@ -22,6 +29,13 @@ export class HomeCommunityComponent implements OnInit, OnDestroy {
         this.seoService.noIndex(true);
       }
     });
+    this.communitiesService.userManagedCommunities$.subscribe((data: ICommunity[]) => {
+      this.managedCommunities = data;
+    });
+    let organizerMatch = this.managedCommunities.find((cSlug) => cSlug.slug === this.community.slug);
+    if (organizerMatch !== undefined) {
+      this.isOrganizer = true;
+    }
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/public-community/components/home-community/home-community.component.ts]: Optimize this

File: projects/commudle-admin/src/app/feature-modules/public-community/public-community-routing.module.ts [hunk 7]
```diff
@@ -40,6 +41,10 @@ const routes = [
         path: 'speakers',
         component: SpeakersComponent,
       },
+      {
+        path: 'notification',
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/public-community/public-community-routing.module.ts]: notifications

**Review by arpansac — CHANGES_REQUESTED** [review]

**Pull request merged**